R/metab_model-class.R

```diff
@@ -15,11 +16,13 @@ setClass(
   "metab_model",
```

Comment:
Probably not the best place for this comment, but I wonder where we are going to hold the "site" concept and the relevant metadata that goes along with that. For example, we need lat to figure out sunrise and sunset (and potentially lon/lat for utility helpers for average baro, elevation, etc). This will also be tied to our proper handling of the timezones for input data. `LakeMetabolizer` handles lat as an input for the metab functions that use it.